prisma/professores.csv

```diff
@@ -0,0 +1,7 @@
+email;name;password;updated_at;discipline
+ruiter@icomp.ufam.edu.br;Ruiter Braga Caldas;$2b$10$6iAsTEV5rtB/C2Y8jO5gFeLuo2oQUW2JfSgcEgdjeEa0/feys4xTq;'2023-05-19';ICC001
+elaine@super.icomp.ufam.edu.br;Elaine Harada Teixeira de Oliveira;$2b$10$ecx5HL0g90chOF/qrxNIZ.3nvykhZCfGYxP8L.dsSPqIRy0YnHRc6;'2023-05-19';ICC020
+nakamura@icomp.ufam.edu.br;Eduardo Freire Nakamura;$2b$10$TKo0h1Jl8C65uQbrh1l27ezA2aA9XeOssBKaE9U.quLFIAJsCvpeC;'2023-05-19';ICC120
+ana.oran@icomp.ufam.edu.br;Ana Carolina Oran;$2b$10$QHgeMtdU2tQFN1.A9zVI0ePDbJRaAPDiDEjQg3QMWeIlq/wTFSAEe;'2023-05-19';ICC002
+rbarreto@icomp.ufam.edu.br;Raimundo da Silva Barreto;$2b$10$oKuVrEHDtdhyC4zgvF7oPOuAIJge0P1Pg5/rqmzp.7nNgITQLKYJO;'2023-05-19';ICC040
+moises@icomp.ufam.edu.br;Moises Gomes de Carvalho;$2b$10$YXMzplioTnjsbQwZ0M7atesavxeDpc8A7VslkmZdVPqiUbVpiXaBi;'2023-05-19';ICC020
```

Comment:
Remove esse arquivo daqui, amg. Não to achando muito segura deixar no github o login dos professores.